src/components/map/MapService.js

```diff
@@ -370,6 +370,19 @@ goog.require('ga_urlutils_service');
           layers: params.LAYERS,
           format: 'image/png',
           service: 'WMS',
+          version: '1.1.1',
+          request: 'GetMap',
+          srs: 'EPSG:4326',
+          bbox: '{westProjected},{southProjected},' +
+                '{eastProjected},{northProjected}',
+          width: '256',
+          height: '256',
+          styles: 'default',
+          transparent: 'true'
+        } || {
+          layers: params.LAYERS,
+          format: 'image/png',
+          service: 'WMS',
           version: '1.3.0',
           request: 'GetMap',
           crs: 'CRS:84',
```

Comment:
Yes but sometimes when you ask for `1.3.0` the wms service returns `1.1.1` .  It's a failure of the WMS but we need to handle that.